libraries/SPI/SPI.cpp

```diff
@@ -38,37 +38,88 @@ SPIClass::SPIClass() {
 }
 
 void SPIClass::begin() {
-    pinMode(SCK, SPECIAL);  ///< GPIO14
-    pinMode(MISO, SPECIAL); ///< GPIO12
-    pinMode(MOSI, SPECIAL); ///< GPIO13
+	begin(SPI_PINS_HSPI);
```

Comment:
The file used to be formatted with 4 spaces, could you please adjust indentation in your changes?